src/Template/Element/Form/children.twig

```diff
@@ -39,11 +39,16 @@
                             <span class="has-text-size-smaller prop-id"><: related.id :></span>
                         </header>
 
-                        <div class="thumbnail" v-if="related.meta.thumb_url">
-                            <figure>
+                        <div class="thumbnail" v-if="related.type === 'images'" >
```

Comment:
Once bedita/bedita#1632 is merged we can rely on `related.meta.thumb_url` as the only condition, no need to check for `images` of `videos` type.